Formula/snort.rb

```diff
@@ -43,7 +44,7 @@ def install
     inreplace "cmake/FindLuaJIT.cmake", " -pagezero_size 10000 -image_base 100000000\"", "\""
 
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DENABLE_STATIC_DAQ=OFF", "-DENABLE_TCMALLOC=ON"
+      system "cmake", "..", *std_cmake_args, "-DENABLE_TCMALLOC=ON"
```

Comment:
No idea what is better `ENABLE_STATIC_DAQ` value.

However, by changing this, users can use commands out-of-box without `--daq-dir <path/to/homebrew/daq>`.

There may be a way to pre-config this for dynamic daq, but I am not sure what it is.